templates/users.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-all",
+    "namespaced_identifiers": false,
+    "source": {
+      "operation": "{{@ datatype @}}-list",
+      "supports_since": false,
+      "system": "{{@ system @}}",
+      "trace": {},
+      "type": "rest"
+    },
+    "type": "pipe"
+  },
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-collect",
+    "exclude_completeness": [
+      "{{@ system @}}-{{@ datatype @}}-share"
+    ],
+    "namespaced_identifiers": false,
+    "source": {
+      "dataset": "{{@ system @}}-{{@ datatype @}}-all",
+      "type": "dataset"
+    },
+    "transform": [
+      {
+        "properties": {
+          "primary_key": "id",
+          "rest_system": "{{@ system @}}",
+          "share_dataset": "{{@ system @}}-{{@ datatype @}}-share"
+        },
+        "template": "transform-collect-rest",
+        "type": "template"
+      },
+      {
+        "rules": {
+          "default": [
+            [
+              "copy",
+              "*"
+            ],
+            [
+              "merge",
+              [
+                "apply-hops",
```

Comment:
Never hop in a connector, if there is a need to merge let it happen in the mapping. But as mentioned in the comment in manifest you can probably get away with only collecting the hub users. If there is a reason to collect both let me know and we can talk through the best pattern for that.

Reply:
I've removed the merging now. We still need both "users" and "usersyoutrack". "usersyoutrack" is basically a mapping between the user-ids used in the youtrack api and the user-ids of the "Hub" users. I've added some documentation about the users here: https://github.com/sesam-io/youtrack-connector/tree/IS-15139#about-users